rocAL/rocAL/include/circular_buffer.h

```diff
@@ -93,8 +94,8 @@ class CircularBuffer
     cl_device_id _device_id = nullptr;
 #else
     hipStream_t _hip_stream;
-    int _hip_device_id;
-    hipDeviceProp_t *_dev_prop;
+    int _hip_device_id, _hip_canMapHostMemory;
+    //hipDeviceProp_t _dev_prop;
```

Comment:
please remove the commented-out line here as well.

Reply:
done